src/components/preferences/Preferences.js

```diff
@@ -340,10 +425,39 @@ function Preferences(props) {
         }
         return errors;
     };
+
+    const mapPropsToValues = (bootstrap) => {
+        if (bootstrap === null || bootstrap === undefined) {
+            return {};
+        }
+        if (bootstrap?.preferences) {
+            if (bootstrap?.apps_info?.webhooks[0]) {
+                let hook = bootstrap?.apps_info?.webhooks[0];
+                const selectedTopics = hook?.topics;
+                const reducer = (acc, currVal) => ({ ...acc, [currVal]: true });
+                const topics = selectedTopics?.reduce(reducer, {}, 0, {});
```

Comment:
To avoid `uncontrolled to controlled` form field warnings, it's best for every field in the form to have an initial value.

Maybe it's not an issue with `Switch` or `Checkbox` fields, but because the `Webhooks` component is creating a field for every topic in `webhookTopics`, I would initialize a value for every topic:
```suggestion
                const selectedTopics = hook?.topics;
                const reducer = (acc, currVal) => ({ ...acc, [currVal]: selectedTopics?.includes(currVal) });
                const topics = webhookTopics.reduce(reducer, {});
```

Reply:
I always stumble on that `uncontrolled to controlled` but I keep forgetting the solution. Thanks 👍🏽.